main/contracts/contracts.ts

```diff
@@ -149,3 +171,78 @@ export interface IResponderRef {
      */
     disconnect: () => void;
 }
+
+/**
+ * Base interface for message broker adapters that integrate with external messaging systems
+ */
+export interface IMessageBrokerAdapter<T> {
+    /**
+     * Unique identifier for the adapter
+     */
+    readonly id: string;
```

Comment:
IMHO this should not be on here. The adaptor does not need to know anything about the id that the message broker refers to it as.